bundles/org.openhab.binding.linktap/src/main/java/org/openhab/binding/linktap/internal/LinkTapBridgeHandler.java

```diff
@@ -0,0 +1,634 @@
+    private final Object getConfigLock = new Object();
+
+    private volatile String currentGwId = "";
+    private volatile LinkTapBridgeConfiguration config = new LinkTapBridgeConfiguration();
+    private volatile long lastGwCommandRecvTs = 0L;
+    private volatile long lastMdnsScanMillis = -1L;
+
+    private String bridgeKey = "";
+    private IHttpClientProvider httpClientProvider;
+    private @Nullable ScheduledFuture<?> backgroundGwPollingScheduler;
+    private @Nullable ScheduledFuture<?> connectRepair = null;
+
+    protected ExpiringCache<String> lastGetConfigCache = new ExpiringCache<>(Duration.ofSeconds(10),
+            LinkTapBridgeHandler::expireCacheContents);
+
+    private static @Nullable String expireCacheContents() {
+        return null;
+    }
+
+    @Activate
+    public LinkTapBridgeHandler(final Bridge bridge, IHttpClientProvider httpClientProvider,
+            @Reference DiscoveryServiceRegistry discoveryService, @Reference TranslationProvider translationProvider,
+            @Reference LocaleProvider localeProvider) {
+        super(bridge);
+        this.httpClientProvider = httpClientProvider;
+        this.discoverySrvReg = discoveryService;
+        this.translationProvider = translationProvider;
+        this.localeProvider = localeProvider;
+        this.bundle = FrameworkUtil.getBundle(getClass());
+        TransactionProcessor.getInstance().setTranslationProviderInfo(translationProvider, localeProvider, bundle);
+        WebServerApi.getInstance().setTranslationProviderInfo(translationProvider, localeProvider, bundle);
+        BindingServlet.getInstance().setTranslationProviderInfo(translationProvider, localeProvider, bundle);
+    }
+
+    public String getLocalizedText(String key, @Nullable Object @Nullable... arguments) {
+        String result = translationProvider.getText(bundle, key, key, localeProvider.getLocale(), arguments);
+        return Objects.nonNull(result) ? result : key;
+    }
+
+    private void startGwPolling() {
+        synchronized (schedulerLock) {
+            cancelGwPolling();
+            backgroundGwPollingScheduler = scheduler.scheduleWithFixedDelay(() -> {
+                if (lastGwCommandRecvTs + 120000 < System.currentTimeMillis()) {
+                    getGatewayConfiguration();
+                }
+            }, 5000, 120000, TimeUnit.MILLISECONDS);
+        }
+    }
+
+    private void cancelGwPolling() {
+        synchronized (schedulerLock) {
+            final ScheduledFuture<?> ref = backgroundGwPollingScheduler;
+            if (ref != null) {
+                ref.cancel(true);
+                backgroundGwPollingScheduler = null;
+            }
+        }
+    }
+
+    private void requestMdnsScan() {
+        final long sysMillis = System.currentTimeMillis();
+        if (lastMdnsScanMillis + MIN_TIME_BETWEEN_MDNS_SCANS_MS < sysMillis) {
+            logger.debug("Requesting MDNS Scan");
+            discoverySrvReg.startScan(THING_TYPE_GATEWAY, null);
```

Comment:
This line is making CI fail, see for example https://ci.openhab.org/job/openHAB-Addons/1530/org.openhab.addons.bundles$org.openhab.binding.linktap/console.

@openhab/add-ons-maintainers Can you please take care?

Reply:
See #17493